setup.py

```diff
@@ -1,39 +1,63 @@
-#! /usr/bin/env python3
+from pathlib import Path
 
 from setuptools import find_packages, setup
 
 
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
+with open("datagouv/__init__.py") as f:
```

Comment:
Pour être iso avec la façon de récupérer le README ?
```suggestion
with open(this_directory / "datagouv/__init__.py") as f:
```

Reply:
Done!